README.md

```diff
@@ -64,6 +64,20 @@ in order to run the API service
 uvicorn ols.app.main:app --reload --port 8080
 ```
 
+### Optionally run with podman
+There is an all-in-one image that has the document store included already.
+
+1. Follow steps above to create your config yaml and your API key file(s). 
+2. Place your config yaml and your API key file(s) in a known location (eg:
+tmp/config)
+3. Run the all-in-one-container. Example invocation:
+
+   ```sh
+    podman run -it --rm -v `pwd`/tmp/config:/app-root/config:Z \
```

Comment:
i'm not clear how this works, i think the instructions are a bit too handwavy about how to setup the config.yaml and api keys.

The config.yaml containers filepath refs to the apikeys.  The filepath ref in the config yaml either needs to be absolute (in which case it would need to be `/app-root/config/.....` given that's where you're mounting them here), or it needs to be relative to the working dir of the container which in this case is ???? but it certainly won't be `/app-root/config`.  Given you are setting OLS_CONFIG_FILE to `config/olsconfig.yaml` that would imply the working directory is `/app-root`.

Which would mean the api key refs in the config.yaml would need to be specified as `config/api-key.txt`

Reply:
to make this somewhat easier on users, you could mount the host path to the same location inside the container.  Then you don't have to worry about the filepath ref in the config.yaml being valid on the host but not valid inside the container, or vice versa.

Reply:
I can improve the instructions, but in this case some folder you create locally (somethingsomething/config) needs to be mounted into the container at `/app-root/config`. Your local config folder should have both your ols config yaml and your keys.

I have to test it, but my assumption is that the path to the API key files in the ols config yaml would just be `config/blah.txt`. Once I test, I'll improve the docs.

I'm not sure what you mean by "mount the host path to the same location inside the container" -- you can't mount anything directly to `/app-root` because it would completely overwrite all the contents of `/app-root`. You have to mount a folder into `/app-root` or into `/`. Podman/Docker do not support mounting files.

Reply:
> in this case some folder you create locally (somethingsomething/config) needs to be mounted into the container at /app-root/config

why does it have to be mounted there, in particular?  The config.yaml can be mounted into the container anywhere you want, you just have to point the OLS_CONFIG_FILE env to the file.

> I'm not sure what you mean by "mount the host path to the same location inside the container" 

I mean that you can tell people to put config.yaml and their keys in a path like "/tmp/olsconfig" on their local host, and then mount it into the container at that same path ("/tmp/olsconfig"), that way the ols config yaml and the file references it contains will be valid both inside the container, and outside the container.  Assuming they use absolute paths in the file references.  For this to work of course the OLS_CONFIG_FILE env var has to be set correctly, but that's always true.

if they use relative paths, then it entirely depends on what OLS considers its working directory to be (which is not the same as which directory the config.yaml is in)

Reply:
I tested the latest push locally, and it works fine with relative paths to the API key files.

If you're passionate about not mounting the config folder into the `/app-root` folder, I will test and update the readme again. LMK.